Source/Scene/Cesium3DTileset.js

```diff
@@ -131,37 +150,108 @@ define([
         };
 
         /**
-         * DOC_TBA
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * Determines if only the tiles from last frame should be used for rendering.  This
+         * effectively "freezes" the tileset to the previous frame so it is possible to zoom
+         * out and see what was rendered.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
          */
         this.debugFreezeFrame = defaultValue(options.debugFreezeFrame, false);
 
         /**
-         * DOC_TBA
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, assigns a random color to each tile.  This is useful for visualizing
+         * what models belong to what tiles, espeically with additive refinement where models
+         * from parent tiles may be interleaved with models from child tiles.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
          */
         this.debugColorizeTiles = defaultValue(options.debugColorizeTiles, false);
 
         /**
-         * DOC_TBA
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, renders the bounding volume for each tile.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
          */
         this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
 
         /**
-         * DOC_TBA
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * When true, renders the bounding volume for each tile's content.
+         * </p>
+         * <p>
+         * When both {@see Cesium3DTileset#debugShowBoundingVolume} and this are true, the tile's
+         * bounding volume is white, and the content's bounding volume is red.  If the content
+         * doesn't have a bounding volume (indicating it is the same volume as the tile), the
+         * tile's bounding volume is blue.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
```

Comment:
The tile's bounding volume is white if it has a content bounding volume, and red otherwise. The content bounding volume is always blue.